src/apis/unselectThumbnailPageNumbers.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Unselect selected thumbnails
+ * @method WebViewer#unselelctThumbnailPageNumbers
```

Comment:
Should be WebViewerInstance.

Reply:
Thanks for catching this, this has been fixed